app/models/zone.rb

```diff
@@ -48,7 +52,33 @@ def find_master_server
     active_miq_servers.detect(&:is_master?)
   end
 
+  class << self
+    private
+
+    def create_maintenance_zone
+      MiqRegion.seed
```

Comment:
This should not be here

Reply:
as I wrote, it is consistent with existing code, it's the same kind of dependency. https://github.com/ManageIQ/manageiq/blob/master/app/models/miq_server.rb#L157

Reply:
hmm... that shouldn't be there either

Reply:
https://github.com/ManageIQ/manageiq/pull/18173

Reply:
I still think this style of programming is not good, but if you insist of it...it influences several totally unrelated specs just because of making assumptions instead of one clear and safe (and idempotent) line of code.